SmartApi/smartWebSocketV2.py

```diff
@@ -0,0 +1,439 @@
+import struct
+import threading
+import time
+import ssl
+import json
+import websocket
+from datetime import datetime, timedelta
+from threading import Timer
+
+class SmartWebSocketV2(object):
+    """
+    SmartAPI Web Socket version 2
+    """
+
+    ROOT_URI = "ws://smartapisocket.angelone.in/smart-stream"
+    HEART_BEAT_MESSAGE = "ping"
+    HEART_BEAT_INTERVAL = 10     # Adjusted to 10s
+    LITTLE_ENDIAN_BYTE_ORDER = "<"
+    RESUBSCRIBE_FLAG = False
+    # HB_THREAD_FLAG = True
+    MAX_RETRY_ATTEMPT = 1
```

Comment:
this should be made configurable and can be taken from init too. default can be kept as 1 .

Reply:
Made the required change